app.rb

```diff
@@ -0,0 +1,131 @@
+require_relative './Classes/student'
+require_relative './Classes/teacher'
+require_relative './Associations/book'
+require_relative './Associations/rental'
+
+ACTIONS = {
+  1 => :list_books,
+  2 => :list_people,
+  3 => :create_person,
+  4 => :create_book,
+  5 => :create_rental,
+  6 => :list_rentals,
+  7 => :break
+}.freeze
+
+class App
+  def initialize
+    puts 'Welcome to School Library App!'
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def options
+    puts ''
+    puts 'Please choose an option by enterin a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
```

Comment:
- [ ] Please I am unable to create a student because of the error in the screenshot below. Kindly checkout the image below and fix the issue. 

![creating a student](https://user-images.githubusercontent.com/65852621/226146033-54419bdb-7e5e-498c-ad71-576924649775.png)